lib/plugins/event_dispatcher/index.node.ts

```diff
@@ -51,19 +51,22 @@ export const dispatchEvent = function(
     },
   };
 
+  const reqWrapper: { req?: http.ClientRequest } = {};
+
   const requestCallback = function(response?: { statusCode: number }): void {
     if (response && response.statusCode && response.statusCode >= 200 && response.statusCode < 400) {
+      reqWrapper.req?.destroy();
       callback(response);
     }
```

Comment:
What is your opinion about this suggestion?
- Do we want to destroy and explicitly free memory outside 200-400?
- I wonder if setting the `reqWrapper.req` to `undefined` will help garbage collection faster. 🤔 

```suggestion
   if (response && response.statusCode && response.statusCode >= 200 && response.statusCode < 400) {
    callback(response);
  }
  reqWrapper.req?.destroy();
  reqWrapper.req = undefined;
```